qatrack/parts/templatetags/part_tags.py

```
@@ -7,8 +7,10 @@
 
 @register.simple_tag
 def part_list_in_storage(s):
-
-    psc = s.partstoragecollection_set.all()
+    try:
+        psc = s.partstoragecollection_set.all()
+    except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)